charts/timed/values.yaml

```diff
@@ -231,6 +231,8 @@ ingress:
 postgresql:
   # -- Enable PostgreSQL for persistence
   enabled: true
```

Comment:
We may want to get rid of that, as a DB is used anyway?

Reply:
it's used in Charts.yaml to activated/deactivate the whole postgresql subchart, i don't think we can get rid of it